migrations.js

```diff
@@ -79,15 +121,18 @@ var runMigrations = function(meta, migrations, callback) {
 
 module.exports = {
   run: function(callback) {
-    getMeta(function(err, meta) {
+    module.exports.get(function(err, migrations) {
```

Comment:
Yeah I think that's right. IIRC when writing this originally it was too hard to mock out reading the migrations dir.